src/main/java/com/synopsys/integration/alert/component/scheduling/SchedulingUIConfig.java

```diff
@@ -50,13 +55,15 @@ public SchedulingUIConfig() {
 
     @Override
     public List<ConfigField> createFields() {
+        final Long countdownMax = TimeUnit.MILLISECONDS.toSeconds(OnDemandTask.DEFAULT_INTERVAL_MILLISECONDS);
+        final ConfigField accumulatorNextRun = CountdownConfigField.create(SchedulingDescriptor.KEY_ACCUMULATOR_NEXT_RUN, LABEL_ACCUMULATOR_NEXT_RUN, ACCUMULATOR_NEXT_RUN_DESCRIPTION, countdownMax);
```

Comment:
Iterate through the providers and create a countdown field per provider accumulator.
The scheduled tasks that are related to the accumulators should implement an interface.  i.e. Accumulator which is in alert-common.  If the provider has an accumulator task then we ask the accumulator for it's name or task name to include when creating the CountdownConfigField.

Reply:
I think these are both good ideas, however.... We don't intend on keeping this page around for a super long time. The idea is to add the task manager component to the UI once that has all been implemented. Things will be done dynamically there.

If we add more providers before that happens, then I think we should make these changes. Otherwise, I think we can keep it this way for now.